src/chains.ts

```diff
@@ -52,6 +47,15 @@ export const NETWORK_DATA = {
       'https://goerli.optimism.io'
     ),
     layer: 2,
+    bridge: '0x4200000000000000000000000000000000000010',
+  },
+  'base-goerli': {
+    id: 84531,
+    name: 'Base Goerli',
+    provider: new ethers.providers.StaticJsonRpcProvider(
+      'https://goerli.base.org',
```

Comment:
Nit: we should add a chain id here so it doesn't have to waste a network request fetching it